### ISSUE TITLE
Number blocked for more than 44 hours.

### ISSUE THREAD

**Issue by vikasmodgil** [cowinapi/developer.cowin#421]
my cowin account has been blocked for more than 44 hours now. The error states that " your account is blocked for exceeding the search limit". I have not used any Bots or extension still my number is blocked.
No one from the technical team is looking into this. 
please provide some resolution.

**Comment by mohit-soral** [comment]
I am facing the same issue. This was mentioned in another issue as well, however other users there are able to access the portal now.
https://github.com/cowinapi/developer.cowin/issues/405

**Comment by sriramcu** [comment]
How many times did you generate an OTP in the day?

**Comment by vikasmodgil** [comment]
> How many times did you generate an OTP in the day?

I don't know the exact number but on Monday I was trying a lot to login so that I could book the slot.  But then I was blocked. Do you know the resolution?

**Comment by sriramcu** [comment]
> > How many times did you generate an OTP in the day?
> 
> I don't know the exact number but on Monday I was trying a lot to login so that I could book the slot.  But then I was blocked. Do you the resolutin?

Nope I have no idea. I was just wondering what I should do to avoid this...some people said they called 1075 and they said that it happened because a user generated 200 OTPs in a day....so I wanted to know if you also generated these many

**Comment by MM1287** [comment]
Error "Your account is blocked", what has to be done next to unblock situation? Still many of family members yet to be vaccinated completely.

**Comment by mehimanshupatil** [comment]
https://timesofindia.indiatimes.com/india/cowin-to-block-users-logging-1k-searches-50-otps-in-24-hours/articleshow/83387752.cms

**Comment by vikasmodgil** [comment]
Got unblocked after 48 hours. They should have informed the citizens before implementing the policy for blocking the users. Because everyone will try to book a slot and for that you need to login. And everone knows that how difficult it is to book a slot for covaxin.
